src/engraving/libmscore/layoutbreak.h

```diff
@@ -35,6 +36,8 @@ class Factory;
 //---------------------------------------------------------
 class LayoutBreak final : public EngravingItem
 {
+    INJECT(engraving, ui::IUiConfiguration, uiConfiguration)
```

Comment:
better use IEngravingConfiguration, add proxy method, Instead of IUiConfiguration directly 
like this https://github.com/musescore/MuseScore/blob/master/src/engraving/internal/engravingconfiguration.cpp#L154